packages/static/kairos-overlay-files/files/etc/systemd/system/systemd-firstboot.service

```diff
@@ -0,0 +1 @@
+/dev/null
```

Comment:
what is this about?

Reply:
Masking the service by linking it to /dev/null which is what systems does when you mask a service.

We don't want the first boot service to start and block boot boys asking stupid questions so we need to mask it.